bootstrap.ps1

```diff
@@ -3,30 +3,32 @@
 
 $ErrorActionPreference = 'Stop'
 
-Push-Location (Join-Path $PSScriptRoot "src/Simulation/CsharpGeneration")
-    .\FindNuspecReferences.ps1
-Pop-Location
+$Env:NATIVE_SIMULATOR_BUILD_CONFIGURATION = "Release"
 
-Push-Location (Join-Path $PSScriptRoot "src/Simulation/Simulators")
-    .\FindNuspecReferences.ps1
+Push-Location (Join-Path $PSScriptRoot "build")
+    .\prerequisites.ps1
 Pop-Location
 
-if ($Env:ENABLE_QIRRUNTIME -eq "true") {
-    Push-Location (Join-Path $PSScriptRoot "src/QirRuntime")
-        .\install-prereqs.ps1
-    Pop-Location
-} else {
-    Write-Host "Skipping installing prerequisites for qir runtime because ENABLE_QIRRUNTIME variable set to: $Env:ENABLE_QIRRUNTIME."
+# Temporary hack until switch qdk build pipeline to use the new build scripts (as a result it will build the native 
```

Comment:
I don't understand why this is necessary...

Reply:
Before this change _bootstrap_ script would create build folders and run cmake to generate build definitions, and _build_ script would run "cmake --build". I placed both commands into the _build_ script to avoid spreading across multiple scripts the knowledge about target folders, etc. Because I don't know whether anything in e2e pipeline relies on the build folders being created I added the call to new build script here. I suppose, I can remove it and see if the pipeline gets unhappy.

Reply:
e2e pipeline does become unhappy. I don't think the pipeline calls the build.ps1 script at all, instead it invokes cmake directly from https://github.com/microsoft/qdk/blob/main/build/steps-native.yml. The yml should be modified to either do both cmake steps or to call build-native-simulator.ps1. To coordinate between the two pipelines, I've added the hack back. After the change is in, we can update qdk to use the new script (explicit cmake), and after that we can remove the hack.